src/WireMock.Net.Abstractions/Admin/Mappings/MatcherModel.cs

```diff
@@ -73,4 +75,10 @@ public class MatcherModel
     /// </summary>
     public MatcherModel? ContentMatcher { get; set; }
     #endregion
+
+    #region XPathMatcher
+
```

Comment:
can you remove the blank line and add some comment, just like the others?

Reply:
Hi Stef,
of course!

Reply:
Done